app/themes/3column/ui/app/root/root.controller.js

```diff
@@ -4,13 +4,17 @@
   angular.module('app.root')
     .controller('RootCtrl', RootCtrl);
 
-  RootCtrl.$inject = ['messageBoardService'];
+  RootCtrl.$inject = ['messageBoardService', 'userService', '$scope'];
 
-  function RootCtrl(messageBoardService) {
+  function RootCtrl(messageBoardService, userService, $scope) {
     var ctrl = this;
-    angular.extend(ctrl, {
-      messageBoardService: messageBoardService,
-      currentYear: new Date().getUTCFullYear()
```

Comment:
Please preserve currentYear, as it is intended to be used for a dynamic copyright notice..